development/cpp/index.rst

```diff
@@ -1,5 +1,5 @@
-Developing in C++
-=================
+Engine development 
```

Comment:
There's a trailing space here (not sure that it matters much, but RST is very touchy about formatting).